README.md

```diff
@@ -1,17 +1,26 @@
-# Stytch.js with React and Express
+# Stytch React Example App
 
-Example app using the Stytch Javascript SDK, React, and Express
+This example app uses the Stytch React component to send magic links and a server to authenticate magic links. This repo contains several implementations of the same server written in Node and Python.
 
-# Getting started
+In order to use this app, you'll need to build the frontend and start the backend server.
+
+## Configure your Stytch credentials
 
-1. Make sure you have node installed: `brew install node`
-1. Install dependencies: `npm install`
 1. Copy the .env_template to your own .env file (`cp .env_template .env`)
 1. Add your Stytch API credentials from the API keys tab in your Stytch dashboard. You'll need to update `STYTCH_PROJECT_ID` and `STYTCH_SECRET` in your `.env` file with your project ID and secret from the dashboard.
 1. From the API keys tab, create a public token and use it for `STYTCH_PUBLIC_TOKEN` in `Login.js`.
-1. Run the app: `npm start`
-1. Open the app in your browser at `localhost:9000`
 
-# Session management
+## Build the client
+
+2. Build the front end files from the client directory. Note: You'll need to rebuild your files to reflect any changes you want to make.
+   ```
+   $ cd client
+   $ npm run build
+   ```
+3. This example app serves the frontend when you run a backend server. Click on the links below Supported Languages to your preferred language to see instructions on how to run the server.
```

Comment:
We should always use 1 for numbering so we don't need to keep track of the numbers.
```suggestion
1. This example app serves the frontend when you run a backend server. Click on the links under Supported server languages to go to the readme for your preferred language.
```